src/spirv.jl

```diff
@@ -27,6 +27,7 @@ Base.@kwdef struct SPIRVCompilerTarget <: AbstractCompilerTarget
     extensions::Vector{String} = []
     supports_fp16::Bool = true
     supports_fp64::Bool = true
+    replace_copysign_f16 = false
```

Comment:
This seems to specific to PoCL that I'd rather keep this in OpenCL.jl, I think. KA.jl can copy it into nanoOpenCL.jl when needed.

Reply:
Ah, `finish_ir!` is what I was looking for